chain/src/sync.rs

```diff
@@ -115,6 +119,11 @@ impl From<CompactBlock> for pb::CompactBlock {
                 Some(cb.quarantined.into())
             },
             slashed: cb.slashed.into_iter().map(Into::into).collect(),
+            fmd_parameters: if cb.fmd_parameters.is_some() {
+                Some(cb.fmd_parameters.unwrap().into())
+            } else {
+                None
+            },
```

Comment:
I think this could be something like:
```suggestion
            fmd_parameters: cb.fmd_parameters.map(Into::into),
```